docs/docs/quickstart.mdx

```diff
@@ -127,40 +130,13 @@ SMTP server the demo uses to show how e.g. email verification works.
 
 ### Clone ORY Kratos and run it in Docker
 
-To get this example working, you will need Git and
-[Docker, and Docker Compose](https://docs.docker.com/get-docker/) installed on
-your system. No other dependencies are required. Before you start, make sure
-that Docker has enough disk space.
-
-:::tip
-
-This tutorial uses Docker-Compose volumes which have reported to run out of disk
-space. Check the remaining disk space using `docker system df`. If the volumes
-are above the 85% threshold,
-[prune old Docker objects](https://docs.docker.com/config/pruning/) before you
-start!
-
-:::
-
-:::tip
-
-If you encounter build errors (e.g. network timeout), make sure that the network
-is running correctly and run `macke docker` again. If the problem persists, feel
-free to [open an issue](https://github.com/ory/kratos/issues/new/choose).
-
-:::
```

Comment:
Looks like my comment got lost somehow in the GitHub UI void :D

This is very important, we have tons of users that mess this up and come asking us for questions "why does docker say out of disk space????" on GitHub, chat, forum. Please revert this.

Reply:
Sounds good! I wasn't aware that was such a common issue 😄